src/Mezzurite.ts

```diff
@@ -124,8 +128,51 @@ export class MezzuritePlugIn implements ITelemetryPlugin{
         {
             customProperties.framework = JSON.stringify(e.Framework);
         }
-        
         this._appInsights.trackPageViewPerformance(event , customProperties);
+
+
+        if (e.SystemTiming){
+            customProperties.systemTiming = JSON.stringify( e.SystemTiming);
+        }
+        if (e.Timing && e.Timing.length > 0) {
+            for (let i = 0; i < e.Timing.length; i++) {
+                let obj = e.Timing[i];
+                let metricType = obj.metricType.toString();
+                let loadTime = obj.value;
+
+                if (obj.data) {
+                    let componentTimes = JSON.parse(obj.data);
+                    for (let j = 0; j < componentTimes.length; j++) {
+                        let ct = componentTimes[j];
+                        let props = {};
+                        let measurements = {};
+                        props["metricType"] = metricType;
+                        let name = undefined;
+                        let id = undefined;
+
+                        if (ct.name) {
+                            let values=ct.name.split(";");
```

Comment:
We updated the format of the component timing object.  the Id, and Name are now two different nodes of the json.

Please update to :

                        props["componentName"] = ct.name;
                        props["id"] = ct.id;
                        props["startTime"] = ct.startTime.toString();
                        props["endTime"] = ct.endTime.toString();
                        measurements["timeToMount"] = ct.untilMount;
                        measurements["componentLoadTime"] = ct.clt;
                        measurements["loadTime"] = loadTime;
                        
                        if (ct.slowResource){
                        measurements["slowestResourceTime"] = ct.slowResource.endTime;
                        props["slowestResourceName"] = ct.slowResource.name;
                        }

Reply:
what version of the mezzurite sdk should I update to?

Reply:
I see the extension does not pull in mezzurite library and assumes it will fire. I think adding a versio check would be wise here.

Reply:
1.0.15   The test page is out of date.